models/quality_measures/intermediate/nqf_0097_medication_reconciliation_post_discharge/quality_measures__int_nqf0097_numerator.sql

```diff
@@ -0,0 +1,101 @@
+{{ config(
+     enabled = var('quality_measures_enabled',var('claims_enabled',var('clinical_enabled',var('tuva_marts_enabled',False)))) | as_bool
+   )
+}}
+
+with denominator as (
+
+    select
+          patient_id
+        , performance_period_begin
+        , performance_period_end
+        , measure_id
+        , measure_name
+        , measure_version
+        , discharge_date
+    from {{ ref('quality_measures__int_nqf0097_denominator') }}
+
+)
+
+, reconciliation_codes as (
+
+    select
+          concept_name
+        , code
+        , code_system
+    from {{ ref('quality_measures__value_sets') }}
+    where lower(concept_name) in (
+        'medication reconciliation post discharge'
+    )
+
+)
+
+, procedures as (
+
+    select
+          patient_id
+        , procedure_date
+        , source_code
+        , source_code_type
+    from {{ ref('quality_measures__stg_core__procedure')}}
+
+)
+
+, reconcilation_procedures as (
+
+    select
+          procedures.patient_id
+        , procedures.procedure_date
+    from procedures
+    inner join reconsilation_codes
+        on procedures.source_code = reconsilation_codes.code
+            and procedures.source_code_type = reconsilation_codes.code_system
+
+)
+
+, qualifying_patients_with_denominator as (
+
+    select
+        denominator.patient_id
+        , denominator.performance_period_begin
+        , denominator.performance_period_end
+        , denominator.measure_id
+        , denominator.measure_name
+        , denominator.measure_version
+        , reconsilation_procedures.procedure_date as evidence_date
+        , 1 as numerator_flag
+    from denominator
+    inner join reconsilation_procedures
```

Comment:
Same note. Need to update CTE reference.
```suggestion
    inner join reconcilation_procedures
```